commit 0fbbf7b743028232d55f22d9174b3632ca663816
Author: dentiny <1910821171@qq.com>
Date:   Mon Jan 9 01:57:19 2023 -0800

    debug

diff --git a/src/utilities.h b/src/utilities.h
index 1f55eff..b47cc00 100644
--- a/src/utilities.h
+++ b/src/utilities.h
@@ -86,21 +86,8 @@
 // correctly when GetStackTrace() is called with max_depth == 0.
 // Some code may do that.
 
-#if defined(HAVE_LIB_UNWIND)
-# define STACKTRACE_H "stacktrace_libunwind-inl.h"
-#elif defined(HAVE__UNWIND_BACKTRACE)
-# define STACKTRACE_H "stacktrace_unwind-inl.h"
-#elif !defined(NO_FRAME_POINTER)
-# if defined(__i386__) && __GNUC__ >= 2
-#  define STACKTRACE_H "stacktrace_x86-inl.h"
-# elif (defined(__ppc__) || defined(__PPC__)) && __GNUC__ >= 2
-#  define STACKTRACE_H "stacktrace_powerpc-inl.h"
-# elif defined(GLOG_OS_WINDOWS)
-#  define STACKTRACE_H "stacktrace_windows-inl.h"
-# endif
-#endif
 
-#if !defined(STACKTRACE_H) && defined(HAVE_EXECINFO_H)
+#if !defined(STACKTRACE_H)
 # define STACKTRACE_H "stacktrace_generic-inl.h"
 #endif
 
@@ -188,7 +175,7 @@ inline T sync_val_compare_and_swap(T* ptr, T oldval, T newval) {
 #endif
 }
 
-void DumpStackTraceToString(std::string* stacktrace);
+GLOG_EXPORT void DumpStackTraceToString(std::string* stacktrace);
 
 struct CrashReason {
   CrashReason() : filename(0), line_number(0), message(0), depth(0) {}
